src/Components/AddContact/AddContact.js

```diff
@@ -27,9 +27,11 @@ function AddContact({ user }) {
   const [phoneNumber, setPhone] = useState('');
   // const [createContact, { loading: mutationLoading, error: mutationError, data }] = useMutation(CREATE_CONTACT);
   const [createContact, { loading: mutationLoading, error: mutationError }] = useMutation(CREATE_CONTACT);
+  const addedContactAlert = () => toast.success(`${firstName} ${lastName} contact information has been added`);
```

Comment:
So for future reference, can we just do `toast.whatever` to indicate the color/type of pop-up alert?